index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio </title>
+   <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
+    <link rel="stylesheet" href="style.css">
+  </head>
+      <body>
+      <div class="flex-container">
+        <div class="item1">
+          <header class="header">
+            <nav>
+            <p class="time"> 9:14 <i class='fas fa-wifi'></i></p>
+            <button class="button"><a href="#">Logo</a></button> 
+            <a href="#"><i class="fa fa-bars"></i></a>
+            </nav>
+         </header>
+        </div>
+        <div class="item2">
+          <section>
+            <h1 class="mariane">I'm Mariane<br>
+              Glad to see<br>
+             you!<br>
+           </h1>
```

Comment:
- [ ] As this is something caused by changes you made, I have realized that the font size used for the h1 does not match the figma design. Kindly use the correct font size as provided in Figma.

**This is how I am seeing it right now:**

![image](https://user-images.githubusercontent.com/86318284/198291758-6493b884-a51c-490b-917e-3cc4665ecdd4.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/198291861-17e448b8-7d53-48e2-b231-3ca3ca18a69a.png)